mlflow/keras.py

```diff
@@ -144,24 +158,42 @@ def log_model(keras_model, artifact_path, conda_env=None, **kwargs):
     >>>   mlflow.keras.log_model(keras_model, "models")
     """
     Model.log(artifact_path=artifact_path, flavor=mlflow.keras,
-              keras_model=keras_model, conda_env=conda_env, **kwargs)
+              keras_model=keras_model, conda_env=conda_env, custom_objects=custom_objects, **kwargs)
+
 
+def _save_custom_objects(path, custom_objects):
+    import cloudpickle
+    custom_objects_subpath = _CUSTOM_OBJECTS_SAVE_PATH
```

Comment:
Nit: We can just use the module-level variable directly - no need to assign it to a function-local variable.